packages/vant/src/swipe/Swipe.tsx

```diff
@@ -457,7 +457,7 @@ export default defineComponent({
 
     watch(count, () => initialize(state.active));
     watch(() => props.autoplay, autoplay);
-    watch([windowWidth, windowHeight], resize);
+    watch([windowWidth, windowHeight], () => nextTick().then(resize));
```

Comment:
嗯嗯，问题复现了

不过正确的修复方式应该是把 props.width 和 props.height 加入到 watchSource 中，不需要添加额外的 nextTick 来处理

```suggestion
    watch(
      [windowWidth, windowHeight, () => props.width, () => props.height],
      resize
    );
```